### PR TITLE
Change activesupport dependency to use v5

### PR DESCRIPTION
**Pull request by tsuwatch**

### CODE REVIEW

File: dinamo.gemspec [hunk 1]
```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "activesupport", "~> 4.2.4"
+  spec.add_dependency "activesupport", ">= 4.2.4"
```
Comment (namusyaka) [inline dinamo.gemspec]: Why don't you explicitly specify version number?

Comment (tsuwatch) [inline dinamo.gemspec]: What do you mean?
So activesupport can only be used up to 4.3.

Comment (namusyaka) [inline dinamo.gemspec]: You said `Change activesupport dependency to use v5`

Comment (tsuwatch) [inline dinamo.gemspec]: Sorry, I see.
I wanted to say `Change activesupport dependency to can use v5`

**Comment by namusyaka** [comment]
Nice 👍 
I'd like to give you a commit bit.

**Comment by namusyaka** [comment]
@tsuwatch I've just added you as a collaborator to this project.
Please try to merge your commit yourself.

**Comment by tsuwatch** [comment]
Thanks @namusyaka , merged.

**Pull request merged**